src/modules/contentful/touts/contentfulTout.base.js

```diff
@@ -13,22 +13,48 @@ class BaseContentfulTout extends Component {
         description,
         heroButtons,
         media
-      }
+      },
+      displayTitle,
+      searchTerm,
+      productsFound
     } = this.props
-    return (
-      <ContentfulRenderer {...media}>
-        <div className='roa-tout-overlay'>
-          <MirageMarkdown>{description}</MirageMarkdown>
-          <div className='roa-tout-buttons'>
-            {heroButtons && heroButtons.map((button) => {
-              return(
-                <ContentfulRenderer {...button} key={button.sys.id} />
-              )
-            })}
+
+    // If searchTerm or displayTitle present, do not render hero image
+    if (searchTerm) {
+      let searchTitle = `You searched ‘${searchTerm}’`
+      if (!productsFound) searchTitle = `Aw, shucks! 0 results for your search ‘${searchTerm}.’`
+
+      return (
+        <div className='default-shop-header'>
+          <div className='default-shop-header-title-wrapper'>
+            <h1 className='default-shop-header-title default-shop-header-title-search'>{searchTitle}</h1>
           </div>
         </div>
-      </ContentfulRenderer>
-    )
+      )
+    } else if (displayTitle) {
+      return (
+        <div className='default-shop-header'>
+          <div className='default-shop-header-title-wrapper'>
+            <h1 className='default-shop-header-title'>{displayTitle}</h1>
```

Comment:
we could avoid this big `else if` block if we figure out the title (whether search, or display) before this and then set that variable to whatever it should be so we only render this block without an extra `if else` 

if that doesnt make sense let me know i can probably explain it better in person. basically `searchTitle` above and this `displayTitle` could be grabbed from one variable that would be set beforehand (before the return ()`

Reply:
TY, makes sense!